app/services/id_token_builder.rb

```diff
@@ -9,21 +9,22 @@ def initialize(identity, custom_expiration: nil)
   end
 
   def id_token
-    payload = {
-      iss: root_url,
-      aud: identity.service_provider,
-      sub: identity.uuid,
-      acr: acr,
-      nonce: identity.nonce,
-      jti: SecureRandom.urlsafe_base64,
-    }.merge(id_token_timestamp_values)
-
-    JWT.encode(payload, RequestKeyManager.private_key, 'RS256')
+    JWT.encode(jwt_payload, RequestKeyManager.private_key, 'RS256')
   end
 
   private
 
-  def id_token_timestamp_values
+  def jwt_payload
+    OpenidConnectUserInfoPresenter.new(identity).user_info.
+      merge(
```

Comment:
looks like there are 3 merges happening here:

`OpenidConnectUserInfoPresenter.new(identity).user_info`

and 

```
acr: acr,
nonce: identity.nonce,
aud: identity.service_provider,
jti: SecureRandom.urlsafe_base64
```

and 

`id_token_timestamp_claims`

Can we have the list of attrs a name (put in a separate method with a good name) so that `jwt_payload` is just merging the 3 sets of attrs? I think that would improve readability